simulate/platform_ui_adapter.cc

```diff
@@ -27,6 +27,7 @@ PlatformUIAdapter::~PlatformUIAdapter() {
 
 bool PlatformUIAdapter::RefreshMjrContext(const mjModel* m, int fontscale) {
   if (m != last_model_ || fontscale != last_fontscale_) {
+    // con_.depthMapping =   mjDB_ONETOZERO;
```

Comment:
Setting this parameter prior to calling `mjr_makeContext` allows the user to control `depthMapping` and `depthPrecision`. I would rather add arguments to `mjr_makeContext` like `fontscale` but this would be a breaking API change.